### ISSUE TITLE
Rollup failed to resolve import

### ISSUE THREAD

**Issue by rexkenley** [aleclarson/vite-tsconfig-paths#148]
I am getting a "rollup failed to resolve import" error. Can you please show me what I did wrong?

vite.config.ts
![image](https://github.com/user-attachments/assets/455bac71-6fec-4e2f-bb4d-58fcd976d319)

tsconfig.app.json
![image](https://github.com/user-attachments/assets/426f17f4-0b19-4d0e-9eb4-02f269631265)

tsconfig.json
![image](https://github.com/user-attachments/assets/6897941c-6ec0-46b1-9e29-d0b8f4a3d587)

tsc -b returns ok

When I run vite build, I get the error.
![image](https://github.com/user-attachments/assets/3c85a833-9f83-4d5f-9ec4-f4858714baf7)

**Comment by achimstraussdmb2bcom** [comment]
same problem. My tsconfig:
`{
	"compilerOptions": {
		"composite": true,
		"tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
		"skipLibCheck": true,
		"module": "ESNext",
		"moduleResolution": "bundler",
		"allowSyntheticDefaultImports": true,
		"strict": true,
		"noEmit": true,
		"baseUrl": "./src",
		"paths": {
			"pages/*": [
				"pages/*"
			],
			"contexts/*": [
				"contexts/*"
			],
			"components/*": [
				"components/*"
			],
			"services/*": [
				"services/*"
			],
			"hooks/*": [
				"hooks/*"
			],
			"i18n/*": [
				"i18n/*"
			],
			"operations/*": [
				"operations/*"
			],
			"src/*": [
				"*"
			]
		}
	},
	"types": [
		"vitest/globals"
	],
	"include": ["vite.config.ts"]
}
`

**Comment by codethief** [comment]
I can't get it to work, either, but switching back to TypeScript 5.4.2 doesn't seem to fix it @achimstraussdmb2bcom. FWIW, since I make extensive use of [extends](https://www.typescriptlang.org/tsconfig/#extends) and [TSC project references](https://www.typescriptlang.org/docs/handbook/project-references.html), which has been causing issues in many tools, I also tried setting `tsconfigPaths`' options `projects` / `configNames` explicitly and I also tried `parseNative`. But nothing has worked so far.
